neo/neo.csproj

```diff
@@ -27,7 +27,7 @@
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.Server.Kestrel.Https" Version="2.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.WebSockets" Version="2.2.1" />
-    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.2.6" />
+    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.1.4" />
```

Comment:
```suggestion
    <!-- Check https://github.com/neo-project/neo/pull/535 before upgrade this package -->
    <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="2.1.4" />
```

Reply:
We should add a comment because is something easy to forget

Reply:
@shargon, I was not able to commit this suggestion because of permission in @cloud8little private repo.
However, editing online it was possible.